packages/api/cms-api/src/mail-templates/mail-template.command.ts

```diff
@@ -0,0 +1,29 @@
+import { CreateRequestContext, MikroORM } from "@mikro-orm/core";
+import { Logger } from "@nestjs/common";
+import { Command, CommandRunner } from "nest-commander";
+
+import { MailTemplateInterface, MailTemplateService } from "./mail-template.service";
+
+@Command({
+    name: "mail-template:test",
+    description: "test mail template, e.g. ",
+    arguments: "[mailTemplateId] [preparedTestParamsIndex]",
+})
+export class MailTemplateCommand extends CommandRunner {
```

Comment:
This serves only as a basic development support. The final solution should integrate better into the styling workflow like providing some admin ui with auto-refresh or "download to mail-client" support.

Reply:
No, this is also required for any non-prod and especially prod environments for PM's (or customers) to check if mails are properly styled, without the need to create any fancy data-structures or do some lengthy or complicated processes.

and it's only implemented once, but available for all projects. (PM's and QA will love it :-D)